src/plopp/core/utils.py

```diff
@@ -91,3 +91,10 @@ def merge_masks(masks):
     Combine all masks into a single one using the OR operation.
     """
     return reduce(lambda a, b: a | b, masks.values())
+
+
+def coord_to_string(coord):
+    out = value_to_string(coord.values)
```

Comment:
What does this do to long coords?

Reply:
It's only designed to take in either one or two values, in the case of bin edges.
But you are right that I should make that more clear, either in the function name or in a docstring